assets/charts/components/azure-arc-lokomotive/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: azure-arc-onboarding
```

Comment:
we have `azure-arc`, `azure-arc-onboarding` and `azure-arc-lokomotive`. I'd be consistent.

Reply:
renaming the component and helm chart to `azure-arc-onboarding` as it fits the description of the component.

Cannot use the name of the component as `azure-arc` as we name the helm release as the name of the component. This created conflict as Azure Arc itself install some helm charts under the release name `azure-arc`, so if it finds the `azure-arc` release present, it starts looking for config resources as if its an existing cluster on boarded Azure Arc.

[Here](https://github.com/kinvolk/lokomotive/blob/96f755ca3e4111f200022378d3ea7266507e7558/pkg/components/util/install.go#L74) we set the `releaseName` of `helmAction` struct as the name of the component and [here](https://github.com/kinvolk/lokomotive/blob/96f755ca3e4111f200022378d3ea7266507e7558/pkg/components/util/install.go#L96) we actually set the release name to the above `releaseName` field.

Reply:
>Cannot use the name of the component as azure-arc as we name the helm release as the name of the component. This created conflict as Azure Arc itself install some helm charts under the release name azure-arc, so if it finds the azure-arc release present, it starts looking for config resources as if its an existing cluster on boarded Azure Arc.

But this release should be in another namespace, right? And it's still being found by onboarding process?

If so, I'd suggest `azure-arc-onboarding` name for chart and release, and just `azure-arc` for component name. Or maybe even `azure-arc-onboarding` for all?

Reply:
yes that's what I mentioned in the beginning of the comment.

`azure-arc-onboarding` as the name of the component, namespace and the helm chart.